src/vma/sock/sockinfo_udp.cpp

```diff
@@ -582,7 +582,7 @@ int sockinfo_udp::connect(const struct sockaddr *__to, socklen_t __tolen)
 		si_udp_logdbg("bound to %s", m_bound.to_str());
 		in_port_t src_port = m_bound.get_in_port();
 
-		if (TRANS_VMA != find_target_family(ROLE_UDP_CONNECT, m_connected.get_p_sa(), m_bound.get_p_sa())) {
+		if (TRANS_VMA != find_target_family(ROLE_UDP_SENDER, m_connected.get_p_sa(), m_bound.get_p_sa())) {
```

Comment:
I find no reason for calling ROLE_UDP_SENDER under sockinfo_udp::connect()
The original role (ROLE_UDP_CONNECT) seems correct